fusion_docs/guide.mdx

```diff
@@ -6,14 +6,15 @@ title: User guide
 
 Fusion is a virtual, lightweight, distributed file system designed to optimise the data access of Nextflow data pipelines.
 
-Fusion enables seamless filesystem I/O to cloud object stores via a standard POSIX interface resulting in simpler
-pipeline logic and faster, more efficient pipeline execution.
+Fusion enables seamless filesystem I/O to cloud object stores via a standard POSIX interface resulting in simpler pipeline logic and faster, more efficient pipeline execution.
+
+:::note
+Fusion is licensed by Seqera and can be used in Seqera Platform or integrated directly with Nextflow.
```

Comment:
How about: "Fusion requires a license, provided by Seqera, which enables usage within Seqera Platform or directly with Nextflow."

Reply:
Is the plan having a free usage quota that's essentially unlicensed? I think this wording will keep potential users even from evaluate it.

Reply:
Good point, we can incorporate that into something like – 

"Fusion permits trial/evaluation usage for free but production usage requires a license.  Licenses are provided by Seqera and enable usage within Seqera Platform or directly with Nextflow."

Gets a bit wordy but I think it gets the points across.